### DISCUSSION TITLE
System option for interface theme & accessibility

### DISCUSSION THREAD

**Discussion by mkandan** [supabase/supabase#13691] [Feature Requests]
Include a system option in the interface theme.

Additionally (slightly unrelated): Currently the interface theme is "hidden" in Account/Preferences. I've been using supabase for a year and didn't know there was a theme setting for the dashboard. Unlike docs, which have a clear theme selector in the top right:
<img width="1225" alt="image" src="https://user-images.githubusercontent.com/40218068/231841670-da94e704-3a73-4629-9136-a6ca889e214e.png">
The dashboard UI does not.

**Comment by gregnr** [comment]
Hey @mkandan, thanks for the post! We are actually in the middle of refactoring the way we manage theme across the sites. I'll pass this along to @supabase/frontend 😃

**Reply by MildTomato** [reply]
We will almost certainly start getting some consistent navigation across the sites, including a more permanent location to access settings like this. ⚡
